workers/generic-worker/simple_docker.go

```diff
@@ -97,9 +97,8 @@ func RenameCrossDevice(oldpath, newpath string) (err error) {
 }
 
 func defaultTasksDir() string {
-	// assume all user home directories are all in same folder, i.e. the parent
-	// folder of the current user's home folder
-	return filepath.Dir(os.Getenv("HOME"))
+	// ~/tasks seems like a reasonable default for simple and docker engine
+	return filepath.Join(os.Getenv("HOME"), "tasks")
```

Comment:
My personal preference here would be to prefer just `"tasks"`, i.e. relative to the working directory. (This is how `cachesDir` defaults too.)

Reply:
That's a very good point - I'll change it as suggested. Thanks!